src/convertToObject.js

```diff
@@ -1,18 +1,18 @@
 'use strict';
 
 function convertToObject(sourceString) {
-  const result = {};
+  const stylesObject = {};
 
   sourceString
     .split(';')
     .map(el => el.trim())
     .filter(el => el)
     .map(el => el.split(': '))
     .forEach(el => {
-      result[el[0].trimEnd()] = el[1].trimStart();
+      stylesObject[el[0].trimEnd()] = el[1].trimStart();
     });
```

Comment:
niiiiice 🔥


for me, that way was too hard - i used to divide methods chains into some parts)
but it's really cool seein' that in your code